spec/controllers/admin/procedures_controller_spec.rb

```diff
@@ -197,6 +197,19 @@
 
         it { expect(flash[:notice]).to be_present }
       end
+
+      context 'when procedure is correctly saved' do
+        let!(:gestionnaire) { create(:gestionnaire, email: admin.email) }
+
+        before do
+          post :create, params: { procedure: procedure_params }
+        end
+
+        describe "admin can also instruct the procedure as a gestionnaire" do
+          subject { Procedure.last }
+          it { expect(subject.gestionnaires).to include(gestionnaire)  }
```

Comment:
```suggestion
          it { expect(subject.gestionnaires).to include(gestionnaire) }
```

(après lecture du linter)

Reply:
merci :-)